vision/src/autogluon/vision/predictor/predictor.py

```diff
@@ -236,6 +244,9 @@ def fit(self,
         searcher = kwargs['hyperparameter_tune_kwargs']['searcher']
         max_reward = kwargs['hyperparameter_tune_kwargs']['max_reward']
         scheduler_options = kwargs['hyperparameter_tune_kwargs']['scheduler_options']
+        # deep copy to avoid inplace overwrite
+        train_data = copy.deepcopy(train_data)
+        tuning_data = copy.deepcopy(tuning_data)
```

Comment:
This will double memory usage. Can create problems on large datasets.

Reply:
I tested 1M image dataset cost 192M memory for required columns

Reply:
currently it's required as the labels are processed internally